Source/Scene/GlobeTranslucencyFramebuffer.js

```diff
@@ -39,109 +30,72 @@ function GlobeTranslucencyFramebuffer() {
-    !globeTranslucency._packedDepthTexture.isDestroyed() &&
-    globeTranslucency._packedDepthTexture.destroy();
-  globeTranslucency._packedDepthFramebuffer =
-    globeTranslucency._packedDepthFramebuffer &&
-    !globeTranslucency._packedDepthFramebuffer.isDestroyed() &&
-    globeTranslucency._packedDepthFramebuffer.destroy();
+  globeTranslucency._framebuffer.destroyResources();
+  globeTranslucency._packedDepthFramebuffer.destroyResources();
 }
 
-function createResources(globeTranslucency, context, width, height, hdr) {
-  var pixelDatatype = hdr
-    ? context.halfFloatingPointTexture
-      ? PixelDatatype.HALF_FLOAT
-      : PixelDatatype.FLOAT
-    : PixelDatatype.UNSIGNED_BYTE;
-  globeTranslucency._colorTexture = new Texture({
-    context: context,
-    width: width,
-    height: height,
-    pixelFormat: PixelFormat.RGBA,
-    pixelDatatype: pixelDatatype,
-    sampler: Sampler.NEAREST,
-  });
-
-  if (context.depthTexture) {
-    globeTranslucency._depthStencilTexture = new Texture({
-      context: context,
-      width: width,
-      height: height,
-      pixelFormat: PixelFormat.DEPTH_STENCIL,
-      pixelDatatype: PixelDatatype.UNSIGNED_INT_24_8,
-    });
-  } else {
-    globeTranslucency._depthStencilRenderbuffer = new Renderbuffer({
-      context: context,
-      width: width,
-      height: height,
-      format: RenderbufferFormat.DEPTH_STENCIL,
-    });
-  }
-
-  globeTranslucency._framebuffer = new Framebuffer({
-    context: context,
-    colorTextures: [globeTranslucency._colorTexture],
-    depthStencilTexture: globeTranslucency._depthStencilTexture,
-    depthStencilRenderbuffer: globeTranslucency._depthStencilRenderbuffer,
-    destroyAttachments: false,
-  });
-
-  globeTranslucency._packedDepthTexture = new Texture({
-    context: context,
-    width: width,
-    height: height,
-    pixelFormat: PixelFormat.RGBA,
-    pixelDatatype: PixelDatatype.UNSIGNED_BYTE,
-    sampler: Sampler.NEAREST,
-  });
-
-  globeTranslucency._packedDepthFramebuffer = new Framebuffer({
-    context: context,
-    colorTextures: [globeTranslucency._packedDepthTexture],
-    destroyAttachments: false,
-  });
-}
+// var pixelDatatype = hdr
+//   ? context.halfFloatingPointTexture
+//     ? PixelDatatype.HALF_FLOAT
+//     : PixelDatatype.FLOAT
+//   : PixelDatatype.UNSIGNED_BYTE;
+// globeTranslucency._colorTexture = new Texture({
+//   context: context,
+//   width: width,
+//   height: height,
+//   pixelFormat: PixelFormat.RGBA,
+//   pixelDatatype: pixelDatatype,
+//   sampler: Sampler.NEAREST,
+// });
 
 function updateResources(globeTranslucency, context, width, height, hdr) {
-  var colorTexture = globeTranslucency._colorTexture;
-  var textureChanged =
-    !defined(colorTexture) ||
-    colorTexture.width !== width ||
-    colorTexture.height !== height ||
-    hdr !== globeTranslucency._useHdr;
-  if (textureChanged) {
+  if (globeTranslucency._framebuffer.isDirty(width, height, hdr)) {
```

Comment:
Another case where the `isDirty` check could be removed if it's impossible for just one of the framebuffers to be dirty during an update, because neither framebuffer uses shared attachments.